llvm/test/CodeGen/X86/bfloat-constrained.ll

```diff
@@ -0,0 +1,98 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
This file was adapted from its `half` brother. I removed the `f16c` attribute tests because I guess they will not affect `bfloat`. I commented out those `fpext` tests since we can't support it right now.

Reply:
You need to not make it `Custom` for `Subtarget.hasF16C()`